hax/test/integration/testutils.py

```diff
@@ -35,11 +35,11 @@ def trace_call(fn):
     '''
     Decorator that 'logs' the historical sequence of all the methods of the
     given class.
-
     The implementation assumes that
     1. the decorator will decorate methods of a class (not static functions).
     2. the class has field `traces: List[Invocation]`
     '''
```

Comment:
""" ?

Reply:
There are single quote docstring comment  like '''  in most of the modules. We need to convert them into double quote doc string comment and will be taken in next part. The rule pertaining to avoiding usage of ''' must have been be disabled.

Reply:
Will be taken up separately, as per @seagate-sarang-sawant